chart/values.yaml

```diff
@@ -293,6 +293,8 @@ csi:
     tolerations: []
     # -- Set PriorityClass, overrides global
     priorityClassName: ""
+    # -- Prevent modifying the volume mode when creating a PVC from an existing VolumeSnapshot
+    preventVolumeModeConversion: true
```

Comment:
This option doesn't make it clear that this is a snapshot/snapshotController option. Should we put this under `csi.controller.snapshotController`? Also @tiagolobocastro wdyt?

Reply:
Actually it needs to put in two places for the functionality to work i.e in the external provisioner and the snapshot controller. That's why kept it a global csi controller value as both of them are a part of it.

Reply:
I missed that one. LGTM